extensions/vscode/src/eventErrors.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2024 by Posit Software, PBC.
```

Comment:
Similar to HTTP axios errors, creating some tooling to filter and handle incoming SSE error messages that have an error code